### ISSUE TITLE
[1.12.2-1.2.12] Crash on start up

### ISSUE THREAD

**Issue by ChromaPIE** [henkelmax/ultimate-car-mod#7]
**Bug description**
At first I tried to launch the game with 2.0.1 which is not supported anymore and got a crash, then switched to 1.2.12, but got a crash too, and weirdly both are roughly the same. I believe it's not the mod that is to blame. However maybe you could help me out. Really love this mod but can't move to newer versions for some reasons.

**Log files**
Two crash reports:
With 2.0.1: https://paste.ubuntu.com/p/yKjty82DC7/
With 1.2.12: https://paste.ubuntu.com/p/BY8BKn7zqK/

**Versions**
- Minecraft version 1.12.2
- Forge version 2854
- Mod version 1.2.12

**Other mods**
_Listed in the crash reports_

**Comment by henkelmax** [comment]
It seems that the json configs cause this crash.
Have you canged the config?

Please try to delete every config related to this mod.

**Comment by ChromaPIE** [comment]
> It seems that the json configs cause this crash.
> Have you canged the config?
> 
> Please try to delete every config related to this mod.

Never did any changes. The two files in the folder _**car**_ are automatically generated and I do deleted the whole folder before launching with 1.2.12 after I removed 2.0.1.

Just now I deleted the folder and tried again, still got a newly generated folder and, unfortunately, a crash. I'll paste the cfg and the json file.

main.cfg: https://paste.ubuntu.com/p/8kGZGCY8QG/
generator.json: https://paste.ubuntu.com/p/pRQsbKBQ8k/

**Comment by henkelmax** [comment]
I'm sorry, but I tried to set up my old 1.12 workspace, but I can't get it to work with all dependencies and stuff.
It is simply too old and I didn't touch it for over 2 years.

So I can't support this version any more. Minecraft 1.12 is over 3 years old and since then 4 new major versions have been released. I don't have the time to maintain all versions for all of my mods.

**Comment by ChromaPIE** [comment]
Never mind. I shifted the way of thinking because I'm pretty sure that there was something wrong with the files in the **_car_** folder, so I simply created another pure instance, installed Forge and made it launch with only Ultimate Car Mod. Luckily the game successfully launched, proving that the mod itself it totally okay. Then I checked the folder and found two new files that I've never seen: **car.json** and **fuel_station.json**, so it must be something in my commonly-used instance, which containing nearly 300 mods, that prevented these two json files from correctly generated. Anyway thanks for your kind help.
